test/tla/TestHash2.tla

```diff
@@ -0,0 +1,32 @@
+--------------- MODULE TestHash2 --------------
+\* Run Apalache to check the invariant for all combinations of hashes
+\*
```

Comment:
`
\* This is an interesting example that appeared in the TLA+ user group:
\*(https://groups.google.com/g/tlaplus/c/cD1bhMcHxqA).
`